impeller/renderer/backend/gles/blit_pass_gles.h

```diff
@@ -50,6 +50,15 @@ class BlitPassGLES final : public BlitPass {
                                     size_t destination_offset,
                                     std::string label) override;
 
+  // |BlitPass|
+  bool OnCopyBufferToTextureCommand(std::shared_ptr<DeviceBuffer> source,
```

Comment:
I think we should make the source a buffer view. That way we can specify ranges in the source. Also, you'll be able to get rid of source_offset which pretty much does the same thing.

Reply:
Done